assets/js/steps/bibbox.js

```diff
@@ -12,29 +12,32 @@ import NavBar from './components/navbar';
 import CheckOutItems from './check-out-items';
 import PropTypes from 'prop-types';
 import MachineStateContext from '../context/machine-state-context';
-
+import BibboxSounds from './utils/bibbox-sounds';
 /**
  * App. The main entrypoint of the react application.
  *
  * @param boxConfigurationInput
  *   The configuration of the bibbox.
  * @param machineStateInput
  *   The state of the app.
+ * @param reservedBookInput
+ *   The last reserved book handed in.
  * @param actionHandler
  *   Callback on requested state change.
  *
  * @return {*}
  * @constructor
  */
-function Bibbox({ boxConfigurationInput, machineStateInput, actionHandler }) {
+function Bibbox({ boxConfigurationInput, machineStateInput, reservedBookInput, actionHandler }) {
```

Comment:
I don't think I understand why this needs to be placed at this level in the hierarchy.
It seems a bit hackish...